config/initializers/date_formats.rb

```diff
@@ -3,5 +3,7 @@
 Date::DATE_FORMATS[:yearmonth] = "%Y-%m"
 Date::DATE_FORMATS[:humanmonthyear] = "%B %Y"
 Date::DATE_FORMATS[:long] = "%-d %B %Y"
+# Important: the GetIntoTeachingApiClient will use this format - do not change it!
+Time::DATE_FORMATS[:default] = "%Y-%m-%d %H:%M:%S %z"
```

Comment:
I don't think we want this as the default format within the app - I think this should probably be getting fixed in the API client, but failing that the code should specify the serialization format for interactions with the API

Reply:
We don't appear to use the default date format anywhere in the app at the moment so I figured it was safe to change - you could argue that we shouldn't rely on Ruby's default date format in the app anyway and it would be better to override it regardless, as it could change in a future version of Ruby.

I get it feels like an awkward place to be fixing it though. I'll look at if the API could be updated to support this format, or I could look at what monkey patching the ApiClient would look like but that feels iffy to me as well, it would probably require a call like:

```
when 'DateTime'
  date.change(offset: "+00:00").to_s if date.utc?
```

Reply:
I'll drop the second commit here and update the gem to pull in the fix:

https://github.com/DFE-Digital/get-into-teaching-api-ruby-client/pull/28